Yona/Yona/nl.lproj/Localizable.strings

```diff
@@ -97,3 +97,10 @@
 "screendesc2" = "Je doelen delen met de persoon die je vertrouwt";
 "screendesc3" = "Grenzen afspreken en daar anderen in betrekken";
 "screendesc4" = "Controle houden over de tijd die je online bent";
+
+"change-pin" = "Wijzig pincode";
+"privacy" = "Privacy";
+"add-device" = "Device toevoegen";
+"delete-user" = "Unsubscribe";
+"deleteusermessage" = "This is your latest device with private key and your profile and data will be remove from the yona server, please type OK below.";
```

Comment:
The dutch translation is not yet provided from YONA, so i put english text here. Will update it when we get the text from YONA.